### PR TITLE
Add/Remove & Fix loc strings

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: web/static/i18n/en.json [hunk 1]
```diff
@@ -1089,7 +1089,7 @@
   "user.settings.cmds.username_desc": "The username to use when overriding the post.",
   "user.settings.cmds.icon_url_desc": "URL to an icon",
   "user.settings.cmds.trigger_desc": "Word to trigger on",
-  "user.settings.cmds.auto_complete_help": "Show this command in autocomplete list",
+  "user.settings.cmds.auto_complete_help": " Show this command in autocomplete list",
```
Comment (jwilander) [inline web/static/i18n/en.json]: Is this space added on purpose?

Comment (enahum) [inline web/static/i18n/en.json]: @jwilander the one who edited the jsx file should know, remember that I use a [tool](https://github.com/ZBoxApp/i18n4react) to extract the strings directly from the code itself.

**Comment by coreyhulen** [comment]
+1

**Comment by jwilander** [comment]
+1

**Pull request merged**